package Message; 
message UserLogout	 								//ук╨е╣гЁЖ
{ 
   optional int64     uid 		= 1;  //ук╨еID
} 